models/_sector/dex/trades/dex_trades_beta.sql

```diff
@@ -32,12 +38,76 @@ WITH curve AS (
         )
     }}
 )
-SELECT
-    *
-FROM
-    curve
-UNION ALL
-SELECT
-    *
-FROM
-    dexs
\ No newline at end of file
+, as_is_dexs AS (
+    {% for model in as_is_models %}
+    SELECT
+        blockchain
+        , project
+        , version
+        , block_month
+        , block_date
+        , block_time
+        , block_number
+        , token_bought_symbol
+        , token_sold_symbol
+        , token_pair
+        , token_bought_amount
+        , token_sold_amount
+        , token_bought_amount_raw
+        , token_sold_amount_raw
+        , amount_usd
+        , token_bought_address
+        , token_sold_address
+        , taker
+        , maker
+        , project_contract_address
+        , tx_hash
+        , tx_from
+        , tx_to
+        , evt_index
+    FROM
+        {{ model }}
+    {% if is_incremental() %}
+    WHERE
+        {{ incremental_predicate('block_time') }}
+    {% endif %}
+    {% if not loop.last %}
+    UNION ALL
+    {% endif %}
+    {% endfor %}
+)
+
+{% set cte_to_union = ['curve', 'as_is_dexs', 'dexs'] %}
```

Comment:
nice usage on the union of CTEs here

minor, but can you make the list one value per line?

```suggestion
{% set cte_to_union = [
    'curve'
    , 'as_is_dexs'
    , 'dexs'
    ] 
%}
```